hash.c

```diff
@@ -3995,50 +3995,19 @@ rb_hash_update_block_i(VALUE key, VALUE value, VALUE hash)
 
 /*
  *  call-seq:
- *    merge! -> self
- *    merge!(*other_hashes) -> self
- *    merge!(*other_hashes) { |key, old_value, new_value| ... } -> self
+ *    update(*other_hashes) -> self
+ *    update(*other_hashes) { |key, old_value, new_value| ... } -> self
+ *    update -> self
```

Comment:
Ditto here, we don't need to have a signature for no arguments because `update(*other_hashes)` encompasses that.

Reply:
Removed.